src/kv1/read.rs

```diff
@@ -308,6 +308,56 @@ where
     Ok(())
 }
 
+fn make_trip_properties(
+    map_vj_accs: HashMap<String, HashSet<Accessibility>>,
+    collections: &mut Collections,
+) -> Result<()> {
+    let mut map_acc_vjs: BTreeMap<Availability, Vec<String>> = BTreeMap::new();
+    for (vj_id, acc) in map_vj_accs {
+        let avaibility = {
+            if acc.len() == 1 {
+                match acc.iter().next() {
+                    Some(acc) if *acc == Accessibility::Accessible => Availability::Available,
+                    Some(acc) if *acc == Accessibility::NotAccessible => Availability::NotAvailable,
+                    _ => Availability::InformationNotAvailable,
+                }
+            } else {
+                Availability::InformationNotAvailable
+            }
+        };
+        map_acc_vjs
+            .entry(avaibility)
+            .or_insert_with(Vec::new)
+            .push(vj_id.clone());
+    }
+
+    let mut id_incr: u8 = 1;
```

Comment:
Instead of using an manually incremented integer, why don't we start using real UUID with [`uuid`](https://www.crates.io/crates/uuid) crate? This way you don't have to manually handle the ID uniqueness but rather use a `Uuid::new_v5()` or something similar. Note however that if the final loading in memory is based on a `String` then it will probably increase the needed memory...

Reply:
don't use use id like `<type>:<id>` in general ? it does not invalidate the `uuid` stuff (I don't have a strong opinion on this) but maybe it's less useful then